production/avotes-parser-core/avotes_parser/core/parsing.py

```diff
@@ -201,6 +210,16 @@ def parse_script(encoded_script: str) -> EVMScript:
         i, one_call = _parse_single_call(encoded_script, i)
         calls_data.append(one_call)
 
+    if abi_storage is not None:
+        calls_data = [
```

Comment:
I'm not sure this is the correct approach to deal with cases when ABI is loaded but unavailable for the particular signature. I mean, `calls_data` would be overridden with the empty values.